tests/lib/rules/no-get.js

```diff
@@ -337,6 +337,22 @@ ruleTester.run('no-get', rule, {
         },
       ],
     },
+    {
+      // Nested path inside left side of assignment should not autofix.
+      // Optional chaining is not valid in the left side of an assignment.
+      code: "this.get('foo.bar')[123] = 'hello world';",
+      options: [{ useOptionalChaining: true }],
+      output: null,
+      errors: [
+        {
+          message: makeErrorMessageForGet('foo.bar', {
+            isImportedGet: false,
+            useOptionalChaining: true,
```

Comment:
It doesn't make sense to recommend optional chaining in this case.

Reply:
Will fix...

Reply:
Fixed!